draft-ietf-oauth-token-binding.xml

```diff
@@ -1165,6 +1145,28 @@
           <spanx style="verb">client_access_token_token_binding_supported</spanx>
           metadata parameter).
         </t>
+        <t>
```

Comment:
Consider moving these two new paragraphs down into the section immediately following? It's "7.2.  Binding of Refresh Tokens" (or not binding them) so maybe fits in there.